core/src/main/java/fr/sncf/osrd/api/TimetableCacheManager.kt

```diff
@@ -171,4 +184,65 @@ class TimetableCacheManager(
             return map
         }
     }
+
+    /**
+     * Try to get the cached value from valkey, returns null if the value isn't cached or if an
+     * error happened.
+     */
+    private suspend fun tryGetFromValkey(
+        valkeyConnection: StatefulRedisConnection<String, String>,
+        key: String,
+    ): STDCMRequirements? {
+        try {
+            val async = valkeyConnection.async()
+            val data = async.get(key).await() ?: return null
+            logger.info("valkey cache hit")
+
+            val cbor = Cbor {}
+            val serializer = STDCMRequirements.SerializableMap.serializer()
+            val serializableMap = cbor.decodeFromHexString(serializer, data)
+            return serializableMap.toSTDCMRequirements()
+        } catch (e: Exception) {
+            logger.info("error when fetching valkey cache: ${e.message}")
```

Comment:
Raising the log level would probably be nice (but error is too much indeed IMO):
```suggestion
            logger.warn("error when fetching valkey cache: ${e.message}")
```

Reply:
Fixed in 5dfa7dc6db7076ccaabd161ff3cbfe3a182f4b58